PHPUnit/Extensions/Selenium2TestCase/Driver.php

```diff
@@ -132,6 +132,9 @@ public function curl($http_method,
         if ($info['http_code'] == 404) {
             throw new BadMethodCallException("The command $url is not recognized by the server.");
         }
+        if (($info['http_code'] > 399) && ($info['http_code'] < 500)) {
```

Comment:
Sure, happy to make that change;  Do you mean instead of the range (thus dropping the error handling for 500's that occurs next)?